tests/src/Context/LogicalThreadContextTest.cs

```diff
@@ -17,7 +17,7 @@
 //
 #endregion
 
-#if NET_4_5 || NETSTANDARD1_3
+#if NET_4_5 || NETCOREAPP1_0 || NETCOREAPP2_0
```

Comment:
So far the constant `NETSTANDARD1_3` has been used, here it becomes `NETCOREAPP1_0`, is this intended?